### PR TITLE
Release Candidate: 0.9.2

### PR DESCRIPTION
**Pull request by ashelkovnykov**
Atypical release candidate (since this is the first one); all changes going in were force-pushed earlier when the history of `master` was rewritten.

**Status**: Tested successfully

### CODE REVIEW

**Comment by bonbud-macryg** [comment]
I've built and tested the app to my satisfaction on the no-docker build, but when I try to build with Docker I get this:



Same error before and after deleting all the images in Docker. (Might not be relevant as I'm seeing others have this error on a quick Google, but I did just change `KELVIN=415` to `417` rather than upgrade my fakeships.)

**Comment by ashelkovnykov** [comment]
@bonbud-macryg Hmmm - it works fine on my machine. When searching for info about the error you got, I found [this](https://stackoverflow.com/a/66993669/15397371), which says that this is often not the true error. Could you try it and see if a different error log appears?

**Comment by bonbud-macryg** [comment]
I changed line 139 in `build.sh` to `sudo DOCKER_BUILDKIT=0 docker build --tag ${DOCKER_IMAGE}:${VERSION_FULL} .` and got a massive amount of logging [pasted here](https://pastebin.com/2hZUJp46), after which the app built and tested successfully.

On a second build, with the images `node` and `urbit-chess` having been created in Docker, logs look quite normal.

```
2023/02/05 13:20:00 must use ASL logging (which requires CGO) if running as root
Sending build context to Docker daemon    225MB
Step 1/13 : FROM node:slim
 ---> aa4ec2a8face
Step 2/13 : ENV NODE_ENV=prod
 ---> Using cache
 ---> 14c11576a4a6
Step 3/13 : WORKDIR /app/frontend
 ---> Using cache
 ---> efb9fcbd06fb
Step 4/13 : COPY ["src/frontend/package.json", "./"]
 ---> Using cache
 ---> 77820c08e834
Step 5/13 : RUN npm install --verbose
 ---> Using cache
 ---> 2cbaea0a139c
Step 6/13 : COPY ["src/frontend/", "./"]
 ---> Using cache
 ---> 890391573a46
Step 7/13 : RUN rm package-lock.json
 ---> Using cache
 ---> fa87ec8640b2
Step 8/13 : RUN npm run-script lint
 ---> Using cache
 ---> 026930618a87
Step 9/13 : RUN npm run-script build
 ---> Using cache
 ---> a02dfcf23f56
Step 10/13 : RUN mkdir output
 ---> Using cache
 ---> 5f9f80f00b69
Step 11/13 : RUN cp -rfL html/index.html css js output/
 ---> Using cache
 ---> 7f69b5e015b1
Step 12/13 : WORKDIR /app
 ---> Using cache
 ---> 6a511578231c
Step 13/13 : CMD cp -rfL frontend/output/* output/
 ---> Using cache
 ---> 66155e4d21ce
Successfully built 66155e4d21ce
Successfully tagged urbit-chess:0.9.2
```

On a third build, images still in Docker, removing `DOCKER_BUILDKIT=0` from the script:

```
[+] Building 0.4s (2/3)                                                                              
[+] Building 0.5s (3/3) FINISHED                                                                     
 => [internal] load build definition from Dockerfile                                            0.0s
 => => transferring dockerfile: 37B                                                             0.0s
 => [internal] load .dockerignore                                                               0.0s
 => => transferring context: 2B                                                                 0.0s
 => ERROR resolve image config for docker.io/docker/dockerfile:1                                0.4s
------
 > resolve image config for docker.io/docker/dockerfile:1:
------
failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = error getting credentials - err: exit status 1, out: ``
```

File: src/frontend/package.json [hunk 1]
```diff
@@ -17,6 +17,15 @@
...
       "url": "https://ashelkov.com"
+    },
+    {
+      "name": "~bonbud-macryg"
+    },
+    {
+      "name": "~rovmug-ticfyn"
+    },
+    {
+      "name": "~nordus-mocwyl"
```
Comment (brbenji) [inline src/frontend/package.json:28]: Awe! The feels.

**Review by brbenji — APPROVED** [review]
LGTM
Runs just fine on my machine. I spun up a fresh new fleet of fake ships and built with docker and no-docker.
Actually felt nostalgic looking at the original game.

**Comment by ashelkovnykov** [comment]
@bonbud-macryg You appear to be in good company, at least: https://github.com/moby/buildkit/issues/1271

Some people were able to resolve by building just once with `DOCKER_BUILDKIT=0`, then building as normal. Others were able to resolve by calling `sudo docker pull docker.io/docker/dockerfile:1`, possibly with `DOCKER_BUILDKIT=0`.

Seems to be a MacOS issue - is that accurate for you as well?

**Comment by bonbud-macryg** [comment]
@ashelkovnykov Yeah I'm on macOS Ventura. I tried `dockerfile:1` but this is the only version of the relevant code that builds first (and every) time for me:

```
if [ $DOCKER -eq 1 ]; then
  sudo DOCKER_BUILDKIT=0 docker build --tag ${DOCKER_IMAGE}:${VERSION_FULL} .
  sudo docker build --tag ${DOCKER_IMAGE}:${VERSION_FULL} .
  sudo docker run --rm -v ${FRONTEND_DIR}:/app/output/ ${DOCKER_IMAGE}:${VERSION_FULL}

  # Copy additional src files for frontend
  sudo chown -R ${USER}:${USER} ${FRONTEND_DIR}
else
  (cd "$ROOT_DIR/src/frontend"; npm run build-no-docker)
fi
```

**Pull request merged**